components/common/layout/PageLayout.tsx

```diff
@@ -10,10 +10,11 @@ import OgData from '../OgData'
 interface PageLayoutProps {
   seoData: SanitySeo
   navigationURLs: SanityNavigation[]
-  BackgroundWrapper: React.FC
+  BackgroundWrapper: any
```

Comment:
Why `any` here instead of a React type?

Reply:
@bdougie I somehow missed it, but I have fixed the issue now.